packages/components/src/popover/types.ts

```diff
@@ -148,6 +148,13 @@ export type PopoverProps = {
 	 * @default undefined
 	 */
 	variant?: 'unstyled' | 'toolbar';
+	/**
+	 * Places the popover over the reference rectangle and adjusts its size to
+	 * fit it exactly.
+	 *
+	 * @default false
+	 */
+	overlay?: boolean;
```

Comment:
Isn't that the same as the "placement" prop. In other words, should this be another "placement" instead of a separate prop?

Reply:
It could be, if it's fine to add an extra option to the floating UI options.

Reply:
I agree that having `overlay` as a separate prop would basically "override" whatever behavior was set through `placement`.

Probably adding an extra option to `placement` feels like the correct choice here, even if that means deviating from `@floating-ui`. The best thing here would be to make these changes directly in `@floating-ui`, so that we can keep using `placement` without any changes in `@wordpress/components`

Reply:
If I understand it correctly, `overlay` is a type of placement where the popover completely covers the anchor element: it's placed at the anchor's top-left corner, and resized to exactly match the size of the anchor.

`@floating-ui`'s job is then to maintain the placement when things move, resize and scroll.

It could be a nice contribution to `@floating-ui`, it makes sense to me.

Reply:
I agree with @jsnajdr , the best solution here is to keep `overlay` as an additonal value for the `placement` prop and remove the separate `overlay` prop.